server-provider/src/test/java/org/sonatype/tests/http/server/jetty/impl/JettyServerProviderTest.java

```diff
@@ -102,7 +102,6 @@ public void testError()
     URL url = new URL("http://localhost:" + provider.getPort() + "/error/404/errormsg");
     HttpURLConnection conn = (HttpURLConnection) url.openConnection();
     assertEquals(404, conn.getResponseCode());
-    assertEquals("errormsg", conn.getResponseMessage());
```

Comment:
iirc Jetty stopped supporting the way this was implemented as it violated an RFC though I can't find the ticket